src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,120 @@
+.product-card {
+  width: 200px;
+  height: 408px;
```

Comment:
Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.

Reply:
Andriy write good comment.

It's point 8 from [task checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)